src/index.html

```diff
@@ -11,6 +11,25 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form class="search-big" data-qa="big">
+      <label class="search-big_label">
+        <input
+        type="search"
+        placeholder="Try &quot;Los Angeles&quot;"
+        data-qa="keypress"
+        class="search-big_input"
+      >
+      </label>
+    </form>
+
+    <form class="search-small" data-qa="small">
```

Comment:
```suggestion
    <form class="search search--small" data-qa="small">
```

So we can add all the styles to `search `class and override them using `search--small` for your small form